vnext/Microsoft.ReactNative.sln

```diff
@@ -97,12 +97,12 @@ Global
 		Chakra\Chakra.vcxitems*{c38970c0-5fbf-4d69-90d8-cbac225ae895}*SharedItemsImports = 9
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|ARM = Debug|ARM
 		Debug|x64 = Debug|x64
 		Debug|x86 = Debug|x86
-		Release|ARM = Release|ARM
+		Debug|ARM = Debug|ARM
```

Comment:
Thanks! Visual Studio always messes this order and defaults to `ARM` for command line builds.